webapp/src/components/navigation/navigation.tsx

```diff
@@ -101,23 +102,40 @@ const Container = styled("div")`
   padding-left: 12px;
   font-family: Inter;
   background-color: #fff;
-  border-top: 1px solid #E0E0E0;
+  border-top: 1px solid #e0e0e0;
 `;
 
 const Sheets = styled("div")`
   flex-grow: 2;
   overflow: hidden;
+  overflow-x: auto;
+
+  &::-webkit-scrollbar {
```

Comment:
I don't think we need this at all.

Reply:
I'm curious, why? This is there to hide the scrollbar.

Reply:
For instance this might just break Firefox, Edge and even Chrome (remember that chrome is not webkit anymore, although I suspect they will keep this particular rule).

Rule of thumb: never use browser specific rules unless you absolutely must, and then you have to figure out the rules for all browsers.

Reply:
Thanks for the explanation! Super helpful. I'll make a new PR to remove that part.